### PR TITLE
Explain the Savas name

### PR DESCRIPTION
**Pull request by chrisarusso**
Redmine: http://redmine.savasdev.com/issues/14

Please @kostajh and @AnneTee take a look.

@AnneTee is this the structure of the HTML as well as the SCSS you'd prefer?
Also, is it still okay to use `<i>italics</i>`? It seems that's no longer okay :guitar:

### CODE REVIEW

**Comment by kostajh** [comment]
@chrisarusso I think `<em>` is the preferred nomenclature nowadays

**Comment by kostajh** [comment]
@chrisarusso I like the write up, and think it reads really well. Some suggestions:
1. I would change `What Does Savas Mean?` to `What does Savas mean?`
2. I find reading the centered text to be difficult with so many lines of text together. What about wrapping the last sentence in `<p>` tags, and adding a `<br />` in between the two paragraphs? I was going to suggest left-alignment but that would be a little too ironic since it would throw off the balance of the page :-)

**Comment by kostajh** [comment]
p.p.s. rebase off of `master` (https://github.com/savaslabs/savaslabs.github.io/commit/d6282b320a9ad62cd1acce0924a219476f83348b) to get `scss-lint` in the travis build, so the tests will pass.

File: who-we-are.html [hunk 1]
```diff
@@ -29,7 +29,13 @@
         {% endfor %}
     </div>
 </div>
-
+<div class="name-explanation wrapper">
+    <h3>What Does Savas Mean?</h3>
+    <p>
+        The name Savas derives its route from the French word <i>savoir</i> which means "to know". Savas is also a <a href="http://en.wikipedia.org/wiki/Palindrome" target="_blank" >palindrome</a>,
```
Comment (AnneTee) [inline who-we-are.html]: @chrisarusso "route" should be "root" here

**Comment by AnneTee** [comment]
@chrisarusso `<i>` is still okay to use when it's not possible/efficient to use CSS for font-weight. `<em>` is specifically for [adding emphasis](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em), so it doesn't apply here.

@chrisarusso @kostajh This looks great, but I think one thing that might help overall is making centered, larger text like what Chris added here and the page description up top only span 8 columns (like /what-we-do and /drupal-training. I think this would help with readability.

**Comment by chrisarusso** [comment]
Misspelling rectified...

@AnneTee let's look at your suggestions together Friday

**Comment by chrisarusso** [comment]
I think @kostajh has a point too about capitalization, we should stay consistent across the site, so I'll fix that too.

**Comment by chrisarusso** [comment]
Hey @AnneTee can you please double-check on what I did with the reworking of the 8-column sitch, and if you feel good about it, merge away?!

**Comment by AnneTee** [comment]
@chrisarusso ++++++

**Pull request merged**